gbpservice/tests/contrib/diskimage-create/elements/nfp-reference-configurator/extra-data.d/99-nfp-service

```diff
@@ -1,7 +1,9 @@
 #! /bin/bash
 set -eu
 
-# copy the reference_configurator folder to VM at /root/
+# copy the reference_configurator and pecan_api folders to VM at /root/
 cp -rL ${SERVICE_GIT_PATH}/reference_configurator ${TMP_MOUNT_PATH}/root/
+cp -rL ${PECAN_GIT_PATH}/pecan_api ${TMP_MOUNT_PATH}/root/
```

Comment:
Are we still copying these to /root/ dir? I thought we moved to the /usr/lib/python.... path.